### PR TITLE
allow flags to be defined multiple times

### PR DESCRIPTION
**Pull request by marten-seemann**

### CODE REVIEW

File: interop/run.py [hunk 1]
```diff
@@ -13,10 +13,10 @@ def get_args():
   parser.add_argument('-d', '--debug', action='store_const',
                       const=True, default=False,
                       help='turn on debug logs')
-  parser.add_argument("-s", "--server", help="server implementations (comma-separated)")
-  parser.add_argument("-c", "--client", help="client implementations (comma-separated)")
-  parser.add_argument("-t", "--test", help="test cases (comma-separatated)")
-  parser.add_argument("-r", "--replace", help="replace path of implementation. Example: -r myquicimpl=dockertagname")
+  parser.add_argument("-s", "--server", action="append", help="server implementations")
```
Comment (janaiyengar) [inline interop/run.py]:
```suggestion
  parser.add_argument("-s", "--server", action="append", help="server implementation. This option can be used multiple times.")
```

File: interop/run.py [hunk 2]
```diff
@@ -13,10 +13,10 @@ def get_args():
...
+  parser.add_argument("-c", "--client", action="append", help="client implementations")
```
Comment (janaiyengar) [inline interop/run.py]:
```suggestion
  parser.add_argument("-c", "--client", action="append", help="client implementation. This option can be used multiple times.")
```

File: interop/run.py [hunk 3]
```diff
@@ -13,10 +13,10 @@ def get_args():
...
+  parser.add_argument("-t", "--test", action="append", help="test cases")
```
Comment (janaiyengar) [inline interop/run.py]:
```suggestion
  parser.add_argument("-t", "--test", action="append", help="test case.  This option can be used multiple times.")
```

File: interop/run.py [hunk 4]
```diff
@@ -13,10 +13,10 @@ def get_args():
...
+  parser.add_argument("-r", "--replace", action="append", help="replace path of implementation. Example: -r myquicimpl=dockertagname")
```
Comment (janaiyengar) [inline interop/run.py]:
```suggestion
  parser.add_argument("-r", "--replace", action="append", help="subsitute implementation. Example: -r myquicimpl=dockertagname. This option can be used multiple times.")
```

File: interop/run.py [hunk 5]
```diff
@@ -162,7 +162,7 @@ def _run_testcase(self, server: str, client: str, testcase: testcases.TestCase)
       "CLIENT=" + IMPLEMENTATIONS[client] + " "
       "SERVER=" + IMPLEMENTATIONS[server] + " "
       "REQUESTS=\"" + reqs + "\" "
-      "docker-compose -f ../docker-compose.yml -f interop.yml up --abort-on-container-exit --timeout 1"
+      "docker-compose -f ../docker-compose.yml -f interop.yml up --abort-on-container-exit --timeout 0"
```
Comment (janaiyengar) [inline interop/run.py]: What does this change do?

**Comment by janaiyengar** [comment]
Also, how about --substitute (-b), instead of --replace?

**Comment by marten-seemann** [comment]
Actually, I quite like the comma-separated list for `client` and `server`, to be honest. I think it's a lot more readable once we have a few implementations there. Let's not remove that one.

**Comment by janaiyengar** [comment]
Yeah, I was ambivalent about those. I agree that it might be better as is. Can you make those changes?

**Comment by marten-seemann** [comment]
Closing this PR. Maybe what we currently have is not too bad for now.

**Pull request closed without merging**